test/libethereum/BlockchainTestsFiller/bcValidBlockTestFiller.json

```diff
@@ -889,7 +889,7 @@
         ]
     },
 
-    "ExtraData1024" : {
+    "ExtraData32" : {
         "genesisBlockHeader" : {
```

Comment:
rather than changing the test, make a new one, both could be valid test fillers.